### ISSUE TITLE
CriticalBug - Disks disappear from disk drives

### ISSUE THREAD

**Issue by Loothor** [raoulvdberge/refinedstorage#227]
Hi there...

Our entire inventory is lost (unless there is a way to restore it.) 

`[16:54:24] [Server thread/ERROR] [FML/]: A TileEntity type refinedstorage.tile.TileDiskDrive has throw an exception trying to write state. It will not persist. Report this to the mod author
java.lang.NullPointerException
	at refinedstorage.apiimpl.storage.NBTStorage.writeToNBT(NBTStorage.java:89) ~[NBTStorage.class:?]
	at refinedstorage.tile.TileDiskDrive.write(TileDiskDrive.java:181) ~[TileDiskDrive.class:?]
	at refinedstorage.tile.TileBase.func_189515_b(TileBase.java:121) ~[TileBase.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75820_a(AnvilChunkLoader.java:384) [atj.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75816_a(AnvilChunkLoader.java:175) [atj.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_73242_b(ChunkProviderServer.java:202) [lr.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_186027_a(ChunkProviderServer.java:230) [lr.class:?]
	at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1011) [ls.class:?]
	at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:355) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:636) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_91]`

For the next 30 minutes, this error occurred every few minutes:

`[16:55:56] [Server thread/ERROR] [FML/]: A TileEntity type refinedstorage.tile.TileDiskDrive has throw an exception trying to write state. It will not persist. Report this to the mod author
java.lang.NullPointerException`

Haven't seen it again. Any ideas? Any way to restore the lost 8 64k disks?

**Comment by raoulvdberge** [comment]
Which Refined Storage version are you using?

**Comment by raoulvdberge** [comment]
This shouldn't be happening. Unless you have backups, there is no way to restore your inventory.

Is it possible that there are items from a *removed* mod in your RS system?

**Comment by Loothor** [comment]
Hi there, we were using 0.8.12 - and weird thing is, it happened without anything going on with the server - no reboots, no removed mods. However I can tell you that we set up a quarry to a 'global' ender chest and was pumping items from that and a miner into the RS system. 

We've decided to start new with a 1.7.10 mudpack until 1.10.2 becomes more stable (issues with IC2 tileentity errors as well and cables becoming disconnected). Thanks for all your contributions and keep up the good work!
